user-commands.go

```diff
@@ -303,6 +303,7 @@ type AddServiceAccountReq struct {
 	AccessKey  string          `json:"accessKey,omitempty"`
 	SecretKey  string          `json:"secretKey,omitempty"`
 	Comment    string          `json:"comment,omitempty"`
+	Duration   time.Duration   `json:"duration,omitempty"`
```

Comment:
From an API perspective I think `Expire *time.Time` is more clear, and leave the time calculation to the clients.

Reply:
Ok. For this, the clients need to calculate the time w.r.t the server timezones. The expiration time will be invalidated accordingly if it falls behind the server's time.

^^ @klauspost

Reply:
Time zones shouldn't be a problem. When sending as JSON the time zone is included, so you are sending an absolute time.

The server can validate and convert to UTC.